remoting/etcdv3/client_test.go

```diff
@@ -0,0 +1,366 @@
+package etcdv3
+
+import (
+	"fmt"
+	"net/url"
+	"path"
+	"reflect"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+)
+
+import (
+	"github.com/coreos/etcd/mvcc/mvccpb"
+	"github.com/pkg/errors"
```

Comment:
It has the same name as errors in the standard library. So we need rename to "perrors", eg: perrors "github.com/pkg/errors".   
And check all please.

Reply:
Thx, i had rename github.com/pkg/errors to perrors.